diff --git a/usr/lib/hypnotix/hypnotix.py b/usr/lib/hypnotix/hypnotix.py
index f163646..07dbb78 100755
--- a/usr/lib/hypnotix/hypnotix.py
+++ b/usr/lib/hypnotix/hypnotix.py
@@ -511,7 +511,7 @@ class MainWindow:
         if " " not in string:
             return string
         words = string.split()
-        if word in string:
+        if word in words:
             words.remove(word)
         return " ".join(words)
 
